### PR TITLE
chore(fx-2622): migrates /art-fairs2 => /art-fairs

### PR DESCRIPTION
**Pull request by dzucconi**
Closes: [FX-2622](https://artsyproduct.atlassian.net/browse/FX-2622)

Moves https://www.artsy.net/art-fairs2 to https://www.artsy.net/art-fairs and deletes old fairs apps.

### CODE REVIEW

**Comment by dzucconi** [comment]
Woops — need to add the page tracking. Should probably build a simple checklist for these replatformings.

**Comment by damassi** [comment]
Maybe for now we can add the list of general requirements here: https://github.com/artsy/force/tree/master/src/v2/Apps/Example

File: src/v2/Apps/Example/README.md [hunk 1]
```diff
@@ -10,3 +10,9 @@ It includes:
 - [ ] Testing
 - [ ] Analytics
 - [ ] Mention of adding tests to integrity
+
+## Launch checklist
+
+- [ ] Ensure meta tags are fleshed out and correct (title, description, image, canonical)
+- [ ] If migrating a legacy app, ensure any redirects or route aliases are ported over
+- [ ] Add the new page to [the list of analytics exclusions](https://github.com/artsy/force/blob/master/src/desktop/assets/analytics.ts)
```
Comment (dzucconi) [inline src/v2/Apps/Example/README.md:18]: @damassi anything obvious I'm missing?

Comment (damassi) [reply src/v2/Apps/Example/README.md:18]: We should probably add a line about analytics sign off with Data team

**Review by damassi — APPROVED** [review]
🎉

**Pull request merged**